proxies/sandbox/apiproxy/resources/jsc/ConnectionMetadata.SetRequestHeaders.js

```diff
@@ -26,7 +26,10 @@ const nrlPointers = {
 };
 //-------------------------------------------------------------------//
 
-const permissions = ["audit-dates-from-payload"];
+const permissions = [
```

Comment:
These permissions are going to be applied to everyone using the sandbox environments - I think this will cause issues in the long run as we would need to put up a PR to remove them every time we didn't want permissions, its a unique problem that only crops up in sandbox because we cant use an app due to "apigee". We probably need to think of a different mechanism to trigger permissions in the sandbox